scripts/common/kubernetes.sh

```diff
@@ -61,7 +61,11 @@ function kubernetes_sysctl_config() {
 function kubernetes_install_host_packages() {
     k8sVersion=$1
 
-    logStep "Install kubelet, kubeadm, kubectl and cni host packages"
+    # ALWAYS update crictl: https://listman.redhat.com/archives/rhsa-announce/2019-October/msg00038.html 
+    tar -C /usr/bin -xzf "$DIR/packages/kubernetes/${k8sVersion}/assets/crictl-linux-amd64.tar.gz"
+    chmod a+rx /usr/bin/crictl
```

Comment:
is this necessary? does crictl talk to a daemon? if not can we work on limiting perms?

Reply:
looks the same, never mind

```
ethan@ethan-kurl-docker:~$ ls -l /usr/bin/kubeadm
-rwxr-xr-x 1 root root 39059456 Jan 13 19:26 /usr/bin/kubeadm
ethan@ethan-kurl-docker:~$ ls -l /usr/bin/crictl
-rwxr-xr-x 1 ubuntu ubuntu 26061879 Dec 13  2018 /usr/bin/crictl
```

Reply:
As I understand if crictl does talk to the kubelet and also to containerd. You still need elevated privileges to access the containerd socket though.